### ISSUE TITLE
Cannot read WSRT TOAs

### ISSUE THREAD

**Issue by dlakaplan** [nanograv/PINT#622]
I am trying to read in a `.tim` file with WSRT data along with GBT and others.  PINT fails on the WSRT data:

```---> 17 toas=toa.get_TOAs('2129-04_WSRT.tim')
     18 
     19 # note that I had to change this from BTX -> BT
~/anaconda/envs/astroconda3/lib/python3.6/site-packages/pint-0.5.4+517.g70e04c0-py3.6.egg/pint/toa.py in get_TOAs(timfile, ephem, include_bipm, bipm_version, include_gps, planets, usepickle, tdb_method)
    103             include_gps=include_gps,
    104             include_bipm=include_bipm,
--> 105             bipm_version=bipm_version,
    106         )
    107     if "tdb" not in t.table.colnames:
~/anaconda/envs/astroconda3/lib/python3.6/site-packages/pint-0.5.4+517.g70e04c0-py3.6.egg/pint/toa.py in apply_clock_corrections(self, include_bipm, bipm_version, include_gps)
   1198                     times[jj] += time.TimeDelta(corr[jj])
   1199 
-> 1200             gcorr = site.clock_corrections(time.Time(grp["mjd"]))
   1201             for jj, cc in enumerate(gcorr):
   1202                 grp["mjd"][jj] += time.TimeDelta(cc)
~/anaconda/envs/astroconda3/lib/python3.6/site-packages/pint-0.5.4+517.g70e04c0-py3.6.egg/pint/observatory/topo_obs.py in clock_corrections(self, t)
    229                 )
    230         log.info("Evaluating observatory clock corrections.")
--> 231         corr = self._clock[0].evaluate(t)
    232         for clock in self._clock[1:]:
    233             corr += clock.evaluate(t)
~/anaconda/envs/astroconda3/lib/python3.6/site-packages/pint-0.5.4+517.g70e04c0-py3.6.egg/pint/observatory/clock_file.py in evaluate(self, t, limits)
     79         If limits=='error' an exception will be raised."""
     80 
---> 81         if numpy.any(t < self.time[0]) or numpy.any(t > self.time[-1]):
     82             msg = "Data points out of range in clock file '%s'" % self.filename
     83             if limits == "warn":
~/anaconda/envs/astroconda3/lib/python3.6/site-packages/astropy/utils/misc.py in __getitem__(self, item)
    943     def __getitem__(self, item):
    944         try:
--> 945             return self._apply('__getitem__', item)
    946         except IndexError:
    947             if self.isscalar:
~/anaconda/envs/astroconda3/lib/python3.6/site-packages/astropy/time/core.py in _apply(self, method, format, *args, **kwargs)
   1273         jd1, jd2 = self._time.jd1, self._time.jd2
   1274         if apply_method:
-> 1275             jd1 = apply_method(jd1)
   1276             jd2 = apply_method(jd2)
   1277 
IndexError: index 0 is out of bounds for axis 0 with size 0
```

The input looks like:
```
MODE 1
C WSRT TOAs
C PHASE 1
i                346.250 55939.5064811673982   27.446                 0.00000
i                346.250 55939.5134255791472   23.565                 0.00000
...
```

As far as I can see code `i` is appropriate for WSRT at least in `observatories.py`, but it doesn't seem to have any relevant clock files.

**Comment by dlakaplan** [comment]
PR #621

**Comment by paulray** [comment]
Fixed in #621
